tools/pot/openvino/tools/pot/api/README.md

```diff
@@ -90,10 +30,15 @@ class openvino.tools.pot.Metric()
 ```
 An abstract class representing an accuracy metric.
 
-All subclasses should override the following properties:
-- `value` - returns the accuracy metric value for the last model output.
-- `avg_value` - returns the average accuracy metric value for all model outputs.
-- `attributes` - returns a dictionary of metric attributes:
+All instances should override the following properties:
+- `value` - returns the accuracy metric value for the last model output in a format of `Dict[str, numpy.array]`.
+- `avg_value` - returns the average accuracy metric over collected model results in a format of `Dict[str, numpy.array]`.
+- `higher_better` should return `True` if a higher value of the metric corresponds to a better performance, otherwise, returns `False`. Default implementation returns `True`.
```

Comment:
```suggestion
All instances should override the following properties:
- `value` - returns the accuracy metric value for the last model output in a format of `Dict[str, numpy.array]`.
- `avg_value` - returns the average accuracy metric over collected model results in a format of `Dict[str, numpy.array]`.
- `higher_better` should return `True` if a higher value of the metric corresponds to better performance, otherwise, returns `False`. Default implementation returns `True`.
```